client/src/components/desktop/HomeDesktop.js

```diff
@@ -1,8 +1,10 @@
 import React from 'react';
 import { Textfit } from 'react-textfit';
-import { Col, Row } from 'antd';
+import { Col, Row, Carousel } from 'antd';
 
-const HomeBlock2Desktop = () => {
+import testimonials from '../../data/testimonials';
+
+export const HomeBlock2Desktop = () => {
```

Comment:
Looking at the structure of these components, is it possible to export a component with both of them? So like, export a component called `HomeBlockDesktop` that's a wrapper for `HomeBlock2Desktop` and `HomeBlock3Desktop`, and the same thing for mobile. Then the conditional rendering can just be:

```js
{isMobile ? (
        <HomeBlockMobile>
    ) : (
        <HomeBlockDesktop>
    )
}
```

Reply:
i think it makes more sense if we don't do a wrapper component. If in the future we create a home block `HomeBlock2.5` in between Block 2 & 3 that does not need different mobile/desktop components, having the wrapper would make it less convenient.

Reply:
we get more flexibility in the future if we do 2 diff exports

Reply:
I think that we would still need to add it in between the two somewhere, whether the wrapper component or in the actual Home page component. If we create a wrapper, all the changes can be in a single file, and we never have to touch it externally after.

Reply:
hmm but with current design, we are putting components that need different designs into `mobile` and `desktop` folders. If we do a wrapper, we might have to put a component that does not need different designs into `mobile` and `desktop` folders. This makes their purpose unclear.

Reply:
okay sure that sounds good :), I agree that your way would be much easier then in the future